modules/npm/src/main/scala/lucuma/core/Foo.scala

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSExportTopLevel
+
+@JSExportTopLevel("Foo")
+class Foo extends js.Object
```

Comment:
So I think this will work as:
```js
import { Foo } from 'lucuma-core';
```

If we write `@JSExportTopLevel("Foo", "somewhere")` then it becomes:
```js
import { Foo } from 'lucuma-core/somewhere.js';
```

but not 100% on this :)

Reply:
Note, you can't nest any further than that. See:
- https://github.com/scala-js/scala-js/issues/4742#issuecomment-1518126573

Reply:
Seems right, I'll experiment a bit